### ISSUE TITLE
ChatRTX 4.0 - Sharing Capabilities?

### ISSUE THREAD

**Issue by ttrandynamic** [NVIDIA/trt-llm-rag-windows#85]
I made minor changes to my ChatRTX 3.0 for sharing and after installing the new 4.0. With new interface, everything changes. There doesnt seem to be any ability to share app any longer.

**Comment by anujj** [comment]
Yes, unlike gradio there is no sharing capability in the new version

**Comment by ttrandynamic** [comment]
is there a development plan of achieving the sharing capability similar to gradio? I'm going to be honest, I really like the way ChatRTX functions and pitched this to my company and managed to route my home pc running chatrtx to a small group of employees within my company. We're going through the testing phase and finding solid project management case use (having chatrtx processes our construction contract specs for multiple projects, company standard docs etc). This new update removing the sharing capability, essentially kills the idea of implementing ChatRTX altogether.

**Comment by Alauz** [comment]
I was also deploying 3.0 with sharing to my colleagues previously and we're on the cusp of implementing it as an internal Chatbot with our existing knowledge base to make employee collaboration easier, however 4.0 just wiped out the entire project. Will there be a share button for further development? It's a real pity because this project had massive potential and going back to 3.0 really limits it. Also, the removal of using website links as part of the RAG completely decimates the project as well as our internal knowledgebase is stored on an intranet site. If we could further develop that to allow for web-based login as well it would make a shift to a login-required knowledge base possible if we were to extend it project wise.
